app/src/main/java/bg/zahov/app/data/model/BodyPart.kt

```diff
@@ -1,28 +1,18 @@
 package bg.zahov.app.data.model
-class BodyPartKeys {
-    companion object {
-        const val CORE = "Core"
-        const val ARMS = "Arms"
-        const val BACK = "Back"
-        const val CHEST = "Chest"
-        const val LEGS = "Legs"
-        const val SHOULDERS = "Shoulders"
-        const val OTHER = "Other"
-        const val OLYMPIC = "Olympic"
-    }
-}
 
-enum class BodyPart(val key: String) {
-    Core(BodyPartKeys.CORE),
-    Arms(BodyPartKeys.ARMS),
-    Back(BodyPartKeys.BACK),
-    Chest(BodyPartKeys.CHEST),
-    Legs(BodyPartKeys.LEGS),
-    Shoulders(BodyPartKeys.SHOULDERS),
-    Other(BodyPartKeys.OTHER),
-    Olympic(BodyPartKeys.OLYMPIC);
+import bg.zahov.fitness.app.R
+
+enum class BodyPart(val body: String, val image: Int) {
+    Core("Core", R.drawable.ic_abs),
```

Comment:
use string resources in the enum, use enum directly  or create UI composable function, discuss with @HauntedMilkshake which approach you will take and follow it for consistency
```
@Composable
fun getBodyPartStringResource(bodyPart: BodyPart): String {
    return when(bodyPart) {
        BodyPart.Core -> stringResource(id = R.string.someString)
        BodyPart.Arms -> stringResource(id =  R.string.someString)
        BodyPart.Back -> stringResource(id =  R.string.someString)
            ....
    }
}
```

Reply:
will be using enums with icon resource